tests/integration/dlio_benchmark/setup-env.sh

```diff
@@ -32,7 +33,7 @@ export DLIO_PROFILER_LOG_LEVEL=ERROR
 export DLIO_PROFILER_BIND_SIGNALS=0
 export MV2_BCAST_HWLOC_TOPOLOGY=0
 
-
+mkdir -p ${DYAD_PATH}
```

Comment:
I think we should add -m 775 to be consistent with dyad core and module.

Reply:
FIXED